README.md

```diff
@@ -1,6 +1,6 @@
 # odoc
 
-**odoc** is a documentation generator for OCaml. It reads *doc comments*,
+**odoc** is a documentation generator for OCaml. It reads _doc comments_,
```

Comment:
We all like underscores better! 😛

Reply:
This was done by my prettier, sorry for that :P

Reply:
I'll leave this decision to you :)